### PR TITLE
INTLY-2555 - apicast

### PR DESCRIPTION
**Pull request by pwright**
https://issues.jboss.org/browse/INTLY-2555

### CODE REVIEW

**Comment by jasonmadigan** [comment]
@finp I think this is largely OK, but the section we're missing is @evanshortiss's content that details Integreatly specifics on how to hook up a prod APIcast to an Integreatly cluster

File: modules/ROOT/pages/_partials/gs-deploying-apicast-proc.adoc [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+= Deploying an APIcast API gateway
+
+APIcast is an NGINX based API gateway used to integrate your internal and external API services with the 3scale Platform.
+
+There are two APIcast deployment options:
+
+*Embedded APIcast*: {PRODUCT_SHORT} hosts APIcast in the cluster. In this case, APIcast is already deployed for you, but is not suitable for a production environment.
+
+*Self-managed APIcast*: You can deploy APIcast wherever you want. The self-managed mode is the intended mode of operation for production environments.
```
Comment (evanshortiss) [inline modules/ROOT/pages/_partials/gs-deploying-apicast-proc.adoc]: `intended mode of operation` only applies to 3scale SaaS, so it's not relevant to us IIRC.

File: modules/ROOT/pages/_partials/gs-deploying-apicast-proc.adoc [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+You can perform this procedure as any user.
+
+.Procedure
+
+. Create an access token in 3scale:
+
+.. Log into the {three-scale-name} Portal.
+.. Navigate to the *Account Setting* screen by clicking the gear icon in the top right of the screen.
+.. Select *Tokens* from the *Personal* left hand menu.
+.. Click *Add Access Token* and name the token 'apicast'.
```
Comment (evanshortiss) [inline modules/ROOT/pages/_partials/gs-deploying-apicast-proc.adoc:30]: 👍 I didn't do this in the original steps, but definitely a better practice!

**Comment by evanshortiss** [comment]
@jasonmadigan @finp I can forward the email where this was discussed if necessary.

File: modules/ROOT/pages/_partials/gs-deploying-apicast-proc.adoc (hunk 1 above)
Comment (pwright) [reply modules/ROOT/pages/_partials/gs-deploying-apicast-proc.adoc]: removed all mentions of 'mode'

File: modules/ROOT/pages/_partials/gs-deploying-apicast-proc.adoc [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+= Deploying an APIcast API gateway
+
+APIcast is an NGINX based API gateway used to integrate your internal and external API services with the 3scale Platform.
+
+There are two APIcast deployment options:
+
+*Embedded APIcast*: {PRODUCT_SHORT} hosts APIcast in the cluster. In this case, APIcast is already deployed for you, making it easy to get started with development, but is not suitable for a production environment.
+
+*Self-managed APIcast*: You can deploy APIcast wherever you want. Red Hat recommends using a self-managed deployment for product.
```
Comment (pwright) [inline modules/ROOT/pages/_partials/gs-deploying-apicast-proc.adoc:17]: @evanshortiss  is this true? RH recommend  provisioning an apicast for production?
or just a hangover from their hosted docs?

Comment (evanshortiss) [reply modules/ROOT/pages/_partials/gs-deploying-apicast-proc.adoc:17]: Hangover from hosted/SaaS docs IIRC. Andrew M can clarify though.

**Comment by pwright** [comment]
from talking with Joe, unsure if this is necessary

**Pull request closed without merging**